client/ayon_core/pipeline/create/context.py

```diff
@@ -1401,6 +1401,7 @@ def __init__(
         self._current_folder_path = None
         self._current_task_name = None
         self._current_workfile_path = None
+        self._current_project_settings = None
```

Comment:
Both `self._current_project_settings` and `self._current_task_entity` should be reset to `None` in `reset_current_context`.